src/libraries/Directory.Build.targets

```diff
@@ -82,7 +76,7 @@
   <!-- Libraries-specific binplacing properties -->
   <PropertyGroup>
     <BinPlaceRef Condition="'$(BinPlaceRef)' == '' and ('$(IsReferenceAssemblyProject)' == 'true' or '$(IsRuntimeAndReferenceAssembly)' == 'true')">true</BinPlaceRef>
-    <BinPlaceRuntime Condition="'$(BinPlaceRuntime)' == '' and ('$(IsSourceProject)' == 'true' or '$(IsRuntimeAndReferenceAssembly)' == 'true')">true</BinPlaceRuntime>
+    <BinPlaceRuntime Condition="'$(BinPlaceRuntime)' == '' and '$(IsSourceProject)' == 'true'">true</BinPlaceRuntime>
```

Comment:
I see that you moved `IsRuntimeAndReferenceAssembly to `resolveContract.targets` and changed it a bit. Why did you move it there?

Can you please explain why you removed the condition from `IsNETCoreAppSrc` and `BinPlaceRuntime`?

Reply:
Because `IsruntimeAndReferenceAssembly` is used in a condition in resolveContract.targets and resolveContract.targets is imported before src/libraries/Directory.Build.targets.